project-koku/koku
# Add org unit to user access check

Fixes [COST-2630](https://issues.redhat.com/browse/COST-2630)

Changes proposed in this PR:
* This adds aws.organizational_unit checking to the user-access API to determine if a user has access to AWS features

Testing Instructions:
1. Set your local dev identity with the following 

```
{
        "identity": {
            "account_number": "10001",
            "type": "User",
            "user": {"username": "user_dev", "email": "user_dev@foo.com", "is_org_admin": False, "access": {"aws.organizational_unit": {"read": ["1", "2"]}}},
        },
        "entitlements": {"cost_management": {"is_entitled": "True"}},
    }
```

2. Hit http://127.0.0.1:8000/api/cost-management/v1/user-access/ on main and see "false" for access to AWS
3. Check out this branch and try http://127.0.0.1:8000/api/cost-management/v1/user-access/ again and see that aws access is now appropriately true

---
Additional Context:
Without this, users with RBAC set exclusively on AWS org unit will not be able to use the UI.
